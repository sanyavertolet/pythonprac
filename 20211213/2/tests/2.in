[-710, 4140, 156, -1803, -2687, -6711, -8868, 7045, 1523, 791, 1327, -4380, 6196, 8347, 6748, 5822, 8091, 5506, 4792, 9135, -91, -1427, 2775, -7172, 4120, -1314, -708, -5044, 7305, 7747, 8733, 4917, 7967, -271, 9323, -8472, 9571, -7483, 6161, -4945, 8716, 6851, -4050, -5225, 4617, -7510, -815, 5295, -8599, -3507, 8581, -1801, 1420, 2353, -6502, 8612, -1331, 4783, -8914, -6494, -882, -7903, 3320, -3591, -3403]
