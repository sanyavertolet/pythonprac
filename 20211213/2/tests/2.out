[-8914, -8868, -8599, -8472, -7903, -7510, -7483, -7172, -6711, -6502, -6494, -5225, -5044, -4945, -4380, -4050, -3591, -3507, -3403, -2687, -1803, -1801, -1427, -1331, -1314, -882, -815, -710, -708, -271, -91, 156, 791, 1327, 1420, 1523, 2353, 2775, 3320, 4120, 4140, 4617, 4783, 4792, 4917, 5295, 5506, 5822, 6161, 6196, 6748, 6851, 7045, 7305, 7747, 7967, 8091, 8347, 8581, 8612, 8716, 8733, 9135, 9323, 9571]
